js/lib/helpers/icon.js

```diff
@@ -3,10 +3,11 @@
  *
  * @param {String} name The name of the icon class, without the `fa-` prefix.
  * @param {Object} attrs Any other attributes to apply.
+ * @param {String} prefix The icon class prefix.
```

Comment:
Please fix the docblock to match the implementation. ;)

Reply:
Also, the method description is not completely true anymore.

Reply:
I'm sorry, missed this one. Thanks for the catch.